test/chain/chain_test.dart

```diff
@@ -69,7 +69,6 @@ void main() {
         future.then(expectAsync1((_) {}, count: 0));
       }, expectAsync2((error, chain) {
         expect(error, equals('oh no'));
-        expect(chain, isA<Chain>());
```

Comment:
I can't tell if that changes the intent of the test.

As is, this test is effectively not testing much of anything - we aren't testing that it "blocks errors" at all, and if we do try to test that it would fail.

I don't know if there is a bug in the library, or if this test should be dropped.

Reply:
🤷

Reply:
I'll fix this test in https://github.com/dart-lang/stack_trace/pull/109 and then we can use this PR for the config changes.